packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts

```diff
@@ -6,6 +6,16 @@ import { MessageThreadStyles, SendBoxStylesProps, TypingIndicatorStylesProps } f
 
 const MESSAGE_THREAD_WIDTH = '41.25rem';
 
+/**
+ * @private
+ */
+export const chatScreenContainerStyle = mergeStyles({
+  height: '100%',
+  width: '100%',
+  minHeight: '20rem',
```

Comment:
As the topic header can be hidden through our API, Can this height be smaller?
![image](https://user-images.githubusercontent.com/2684369/141186937-5fec2f5e-7a23-4d7c-a37c-804ce7e8e7ad.png)

I'm thinking the scenario where someone may want to embed this inside a chat window similar to fbs:
![image](https://user-images.githubusercontent.com/2684369/141187169-e509d681-bebc-4f16-bebf-821d1983a58a.png)

(Though seeing them side by side maybe that is a good size?)

Reply:
we can go smaller, but message list will be quite small I feel